src/cci/cas_cci.c

```diff
@@ -693,6 +694,20 @@ cci_disconnect (int mapped_conn_id, T_CCI_ERROR * err_buf)
 
   API_SLOG (con_handle);
 
+  if (con_handle->broker_info[BROKER_INFO_CCI_PCONNECT])
```

Comment:
This is required not to break function flow. 
If this code is not appropriate, we might execute code refactoring.
But I think refactoring should be handled in other issue.